payments-ui-core/src/main/java/com/stripe/android/ui/core/PaymentsTheme.kt

```diff
@@ -42,88 +43,35 @@ data class PaymentsColors(
-        private val fontWeightNormal: Int = FontWeight.Normal.weight
-        private val fontSizeMultiplier: Float = 1.0F
-        val fontFamily: Int = R.font.roboto
-
-        // h4 is our largest headline. It is used for the most important labels in our UI
-        // ex: "Select your payment method" in Payment Sheet.
-        val h4 = TextStyle.Default.copy(
-            fontFamily = FontFamily(Font(fontFamily)),
-            fontSize = (20.0 * fontSizeMultiplier).sp,
-            fontWeight = FontWeight(fontWeightBold),
-        )
-
-        // h5 is our medium headline label.
-        // ex: "Pay $50.99" in Payment Sheet's buy button.
-        val h5 = TextStyle.Default.copy(
-            fontFamily = FontFamily(Font(fontFamily)),
-            fontSize = (16.0 * fontSizeMultiplier).sp,
-            fontWeight = FontWeight(fontWeightMedium),
-            letterSpacing = (-0.32).sp
-        )
-
-        // h6 is our smallest headline label.
-        // ex: Section labels in Payment Sheet
-        val h6 = TextStyle.Default.copy(
-            fontFamily = FontFamily(Font(fontFamily)),
-            fontSize = (13.0 * fontSizeMultiplier).sp,
-            fontWeight = FontWeight(fontWeightMedium),
-            letterSpacing = (-0.15).sp
-        )
-
-        // body1 is our larger body text. Used for the bulk of our elements and forms.
-        // ex: the text used in Payment Sheet's text form elements.
-        val body1 = TextStyle.Default.copy(
-            fontFamily = FontFamily(Font(fontFamily)),
-            fontSize = (14.0 * fontSizeMultiplier).sp,
-            fontWeight = FontWeight(fontWeightNormal),
-        )
-
-        // subtitle1 is our only subtitle size. Used for labeling fields.
-        // ex: the placeholder texts that appear when you type in Payment Sheet's forms.
-        val subtitle1 = TextStyle.Default.copy(
-            fontFamily = FontFamily(Font(fontFamily)),
-            fontSize = (14.0 * fontSizeMultiplier).sp,
-            fontWeight = FontWeight(fontWeightNormal),
-            letterSpacing = (-0.15).sp
-        )
+data class PaymentsShapes(
+    val cornerRadius: Float,
+    val borderStrokeWidth: Float,
+    val borderStrokeWidthSelected: Float,
+)
 
-        // caption is used to label images in payment sheet.
-        // ex: the labels under our payment method selectors in Payment Sheet.
-        val caption = TextStyle.Default.copy(
-            fontFamily = FontFamily(Font(fontFamily)),
-            fontSize = (12.0 * fontSizeMultiplier).sp,
-            fontWeight = FontWeight(fontWeightMedium)
-        )
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+data class PaymentsTypography(
+    val fontWeightNormal: Int,
+    val fontWeightMedium: Int,
+    val fontWeightBold: Int,
+    val fontSizeMultiplier: Float,
+    val xxSmallFont: Float,
+    val xSmallFont: Float,
+    val smallFont: Float,
+    val mediumFont: Float,
+    val largeFont: Float,
+    val xLargeFont: Float,
```

Comment:
I think these should all end with "Size". Also, is there a better type than float? [`TextUnit`](https://developer.android.com/reference/kotlin/androidx/compose/ui/unit/TextUnit) seems better, it's from the Compose library but I think that's fine. WDYT?

Reply:
TextUnit is way better. Since this is not a public variable we can use compose stuff freely here. I pushed the change.